bin/tx-prover/src/proxy/mod.rs

```diff
@@ -68,6 +80,8 @@ impl LoadBalancerState {
             workers.push(Worker::new(worker, connection_timeout, total_timeout).await?);
         }
 
+        WORKER_COUNT.set(workers.len() as i64);
```

Comment:
nit (maybe too much of an edge case): Should this be taken out? For example if the proxy starts with 2 workers out of which 1 was not running correctly, this might give the impression that it was up for a while until the health check kicks in and decrements the metric. Maybe a health check could be forced here or something and we could remove this line, or maybe we just wait until the health check kicks in by itself.

Reply:
It's an interesting case, haha. Even though we don't explicitly call the health check method for each worker, we do create a HealthCheckClient and establish a connection and will return an error if the connection fails. While this doesn't guarantee that the service is __serving__, I believe it's sufficient until the health checker steps in.  What do you think?

Reply:
Ah yes, I forgot that the health check client would error on creation